Examples/PyNetKet/ground_state_iter.py

```diff
@@ -69,29 +68,35 @@ def output(vmc, step):
         ma.save("test.wf")
 
 
-def run_vmc(steps, step_size, diag_shift):
-    op = nk.optimizer.Sgd(step_size)
-    vmc = nk.variational.Vmc(
+def run_vmc(steps, step_size, diag_shift, n_samples):
+    opt = jaxopt.sgd(step_size)
+    # opt = nk.optimizer.Sgd(step_size)
+
+    sr = nk.optimizer.SR(lsq_solver="BDCSVD", diag_shift=diag_shift)
+    sr.store_rank_enabled = False  # not supported by BDCSVD
```

Comment:
maybe this is not related to this PR, but why these options are not just 
```python 
sr.store_rank = True
sr.store_covariance_matrix = True
```

Reply:
We could rename these options, yes. Let's do that in a separate PR.